keras_nlp/models/bert.py

```diff
@@ -61,25 +82,29 @@ class Bert(keras.Model):
     Example usage:
     ```python
     # Randomly initialized Bert encoder
-    encoder = keras_nlp.models.Bert(
+    model = bert.Bert(
```

Comment:
Prefer using `keras_nlp` imports in code examples, e.g. `keras_nlp.models.XYZ`

Reply:
I don't think we can here sadly, because it's inside the library. At least when I tried out `import keras_nlp` it immediately landed in a circular import error. Though maybe there's a way around this?

Reply:
Actually this is my bad, good catch. This is just example code so totally possible. I just copied the code from the test suite to get formatting and didn't fix it.

Reply:
oh haha I see, this is in the code block. my bad!